templates/listings.tpl.php

```diff
@@ -10,11 +10,30 @@
 
 <div id="wpbdp-listings-list" class="listings wpbdp-listings-list list wpbdp-grid <?php echo esc_attr( apply_filters( 'wpbdp_listings_class', '' ) ); ?>">
 	<?php
+	/**
+	 * Filters whether to display the pagination in the listings wrapper or outside of it.
+	 * 
+	 * @since x.x
+	 */
+	$wrapper_pagination = apply_filters( 'wpbdp_listings_wrapper_pagination', true );
```

Comment:
The variable `$wrapper_pagination` and the filter `'wpbdp_listings_wrapper_pagination'` could be clearer.

Could we rename them to `$display_pagination_in_wrapper` and `'wpbdp_display_pagination_in_wrapper'`, or something similar, to better reflect their purpose?